lib/diagnostic.js

```diff
@@ -46,21 +46,31 @@ let person2 = {
 };
 
 // Question 3:  Invoke the `fullName()` method so it returns `Santa Clause`?
-let answer3 = null; // replace null with your response
+let answer3 = {
```

Comment:
This answer misses the point of the exercise.  `person1`'s `fullName()` method needs to be called with `person2` as the context.  This is done by calling `Function`'s `call()` method on `fullName()` with `person2` as the argument.

```javascript
person1.fullName.call(person2);
```